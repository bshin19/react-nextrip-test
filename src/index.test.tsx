import { Test } from "./index";
import { render } from "@testing-library/react";

describe("test", () => {
  it("should exist", () => {
    const { container } = render(<Test />);
    expect(container).toMatchSnapshot();
  });
});
