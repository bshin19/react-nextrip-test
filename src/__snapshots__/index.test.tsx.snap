// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test should exist 1`] = `
<div>
  <div
    class="test-thing"
  />
</div>
`;
